modules/custom/openy_activity_finder/src/OpenyActivityFinderSolrBackend.php

```diff
@@ -146,25 +164,32 @@ public function doSearchRequest($parameters) {
         // Specific case, user selected e.g. '16+'.
         if (strpos($age, '+')) {
           $age = str_replace('+', '', $age);
-          $db_and->addCondition('field_session_min_age', $age, '>=');
-        }
-        // Specific case, user selected less than 18 months (6, 12, 18).
-        else if ($age <= 18) {
-          $db_and->addCondition('field_session_min_age', $age, '>=');
-          $db_and->addCondition('field_session_min_age', $age + 6, '<');
+          $db_and->addCondition('field_session_min_age', $age, '<=');
```

Comment:
See https://github.com/ymcatwincities/ymca/pull/3672/files#diff-1850d6d8e14f9cab18333e1915e6abe7R153 @jediandrey 
There is a bug in AF
Given
Product
From 3 to 7 years
Customer search for 5 years
Product should be displayed.
YGTC code fixes this

Reply:
we have advanced case with + functionality, like 16+ and 55+
also, there could be values as 0 and null for max age field in other projects.

Reply:
16+ and 55+ covered by the code I referenced. @jediandrey

Reply:
@podarok pushed changes as we discussed